### DISCUSSION TITLE
Best strategy to migrate data to fresh Greyhole powered box?

### DISCUSSION THREAD

**Discussion by BigMaxTs** [gboudreau/Greyhole#323] [General]
Good day,

I have an existing nas with ~2Tb of data: photos, files etc, and since it's old I have built a new one and will use GH on it. It's now configured and working fine, so I am thinking on how it would be better to copy data onto a new box so that Greyhole will not get stuck/mad during coping large count of files?

I see 2 basic options now:
1. copy data straight to GH share and let GH do the job
2. disable GH, copy data, enable GH and ??? - this is the question, I have read the wiki here but do not have a clear picture of this scenario.

My GH box is 2 SandyBridge cores/threads, 8Gb RAM, 6 SATA drives in a pool, 2 shares - 3 drives in each share drive group, 1Gig link between old and new box.

As I plan to keep old box as it is (plus it is a synology with proprietary raid config) moving disks from old one to new is not and option.

Thank a lot in advance for any ideas.

Max

**Comment by gboudreau** [comment]
Hi,

There is a wiki article about this here: https://github.com/gboudreau/Greyhole/wiki/ReuseDataDrives  
Basically, copy your files into one of your storage pool drives (folder), in a folder named like your Samba share, then launch a fsck for Greyhole to create extra copies & symlinks as needed, for your files to be protected and become visible through your Samba shares.

**Comment by BigMaxTs** [comment]
Hi,

many thanks, just read the article now once again and for some reason it is all clear now for me. 

Thank you for support, much appreciated.
